app/models/concerns/reporting/monthly_reports/monthly_report_charts.rb

```diff
@@ -203,9 +192,7 @@ def clients_for_report
       @clients_for_report ||= self.class.
         where(project_id: GrdaWarehouse::Hud::Project.viewable_by(user).pluck(:id)).
         in_months(filter.range).
-        for_organizations(filter.organization_ids).
-        for_projects(filter.project_ids).
-        for_project_types(filter.project_type_ids).
+        for_projects(filter.anded_effective_project_ids). # project ids from projects, organizations, project groups and project types
```

Comment:
This inspires me to wonder if adding an optional parameter to select between union or intersection of the filter semantics to get rid of the subclasses in the HUD reports?